app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
```

Comment:
Maybe `true_targets_after_filter`? The name is challenging. I don't think I fully grasp the delete logic though.